index.qmd

```diff
@@ -188,11 +197,12 @@ Notas.
 -   `MinFD`: Diâmetro mínimo de corte, em centímetros
 -   `UpMinFD`: Diâmetro mínimo de corte aprimorado (caso de povoamento excessivamente rico), em centímetros
 -   `MaxFD`: Diâmetro máximo de corte, em centímetros
--   `Aggregative`: Natureza agregadora da espécie para converter árvores isoladas
+-   `Aggregative`: Natureza agregadora da espécie para converter árvores isoladas <!-- not sure converter is the right word... evitar ?
+    Also, I'm not sure we want to explain this as it may take quite some time and it is not central... what do you think?-->
```

Comment:
I I was unsure because it is an input from the table, but I think I'll just drop it.